### PR TITLE
Implement rubocop

### PR DESCRIPTION
**Pull request by will-in-wi**
Run in travis, regenerate todo, and fix a couple of smaller ones.

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -14,4 +15,10 @@ Layout/DotPosition:
   EnforcedStyle: trailing
 
 Style/BracesAroundHashParameters:
-  Enabled: false
\ No newline at end of file
+  Enabled: false
```
Comment (eterry1388) [inline .rubocop.yml]: I vote this one is enabled.  It helps prevent accidental and weird handling of parameters passed into methods.

File: .rubocop.yml [hunk 2]
```diff
@@ -14,4 +15,10 @@ Layout/DotPosition:
...
+
+Layout/AccessModifierIndentation:
+  EnforcedStyle: outdent
+
+Metrics/LineLength:
+  Enabled: false
```
Comment (eterry1388) [inline .rubocop.yml]: Should we enable this and just put on a longer max length?

File: .rubocop.yml [hunk 3]
```diff
@@ -1,6 +1,7 @@
 inherit_from: .rubocop_todo.yml
```
Comment (eterry1388) [inline .rubocop.yml]: How about adding these?

```
Style/SymbolArray:
  Enabled: false

Style/WordArray:
  Enabled: false

Layout/MultilineOperationIndentation:
  EnforcedStyle: 'indented'

Layout/MultilineMethodCallIndentation:
  EnforcedStyle: 'indented'
```

File: .rubocop.yml (hunk 1 above)
Comment (will-in-wi) [reply .rubocop.yml]: I agree, but this was pre-existing, I just added a newline (hence the change). So I'm not going to change strategy in this PR.

File: .rubocop.yml (hunk 3 above)
Comment (will-in-wi) [reply .rubocop.yml]: I added the latter two. The first two are in the todo list in our main repo, so I assume that means we'd like to enable it later.

**Pull request merged**